src/components/application_manager/src/commands/mobile/register_app_interface_response.cc

```diff
@@ -78,11 +78,6 @@ void RegisterAppInterfaceResponse::Run() {
 
   SetHeartBeatTimeout(connection_key(), application->policy_app_id());
 
-  // Default HMI level should be set before any permissions validation, since
-  // it
-  // relies on HMI level.
-  application_manager_.OnApplicationRegistered(application);
```

Comment:
@LuxoftAKutsan , this code is another branches, but not in develop. see [here](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/application_manager/src/commands/mobile/register_app_interface_response.cc)